src/main/java/edu/cornell/kfs/sys/service/impl/ISOCountryServiceImpl.java

```diff
@@ -88,6 +97,21 @@ public String findISOCountryNameByCountryCode(String isoCountryCode) {
         }
     }
 
+    @Override
+    public List<String> findISOCountryCodesByCountryName(String isoCountryName) {
+        if (isBlank(isoCountryName)) {
+            return List.of();
+        }
+        QueryByCriteria criteria = QueryByCriteria.Builder.fromPredicates(
+                PredicateFactory.equalIgnoreCase(CUKFSPropertyConstants.Country.NAME, isoCountryName),
```

Comment:
Table CU_ISO_SH_CNTRY_T contains active rows where country names are in mixed case.
Do we need to make this query case insensitive?
Should leading and trailing white space also be addressed?

Reply:
See my notes on the equivalent area in the `CountryServiceImpl` class. The name-based search is already case-insensitive, but I will add some explicit trimming of the name to deal with potential whitespace.